openshift/template.yaml

```diff
@@ -127,6 +127,11 @@ objects:
               secretKeyRef:
                 name: 3scale
                 key: three_scale_premium_user_key
+          - name: PREMIUM_URL
+            valueFrom:
+              secretKeyRef:
```

Comment:
if you are introducing a new secret, you will need this secret in the vault before you merge this PR

Reply:
I dont think this needs to be a secret. It can be a configurable value in app interface.

Reply:
Since the token is a secret i made this also a secret.

Reply:
there is no need to have this as a secret. please change it

Reply:
Done.